### DISCUSSION TITLE
Update? 4.x

### DISCUSSION THREAD

**Discussion by Simon99de** [lucky-media/landtamic#16] [General]
Hello,
 what about an update to 4.x?
 Or is it already compatible?

**Comment by lokmanm** [comment]
Hey @Simon99de,

The Starter kit should already be compatible. If you have any problems, please don't hesitate to let us know.

However, in the following days, we will release a new update for Landtamic that uses the new Set Picker from Statamic and some further improvements.

Thanks!
